### PR TITLE
manifest.json.jet

### PR DESCRIPTION
**Pull request by gscragg**
add specific manifest for each templated site so it can more easily be configured properly

### CODE REVIEW

**Comment by gscragg** [comment]
THe included submodule could be an issue.

File: README.md [hunk 1]
```diff
@@ -33,7 +33,18 @@ Once you have your logos you can set your local variables for the navbar brand.
...
+
+Set those first and see if it suits. if you need to customise further you will find other variables to tweak button colors and such.
 
 ### Custom css
 Ideally its best to avoid using custom CSS if possible as it could jeopardize updates in the future.
+
+### manifest.json
+The web app manifest is a JSON file that tells the browser about the site and how it should behave when installed on the user's desktop or mobile device. A typical manifest file includes the app name, the icons the app should use, and the URL that should be opened when the app is launched. [more details](https://web.dev/add-manifest/).
+
+Make sure you update the fields in the `manifest.json.jet` to match the sites main colours and logos, and names.
```
Comment (additiverich) [inline README.md:50]: I don't know how to test the contents of this file and wonder if we should expand upon that here.

File: README.md [hunk 2]
```diff
@@ -33,7 +33,20 @@ Once you have your logos you can set your local variables for the navbar brand.
...
+- `$body-bg`
+- `$body-color`
+
+Set those first and see if it suits. if you need to customise further you will find other variables to tweak button colors and such.
 
 ### Custom css
 Ideally its best to avoid using custom CSS if possible as it could jeopardize updates in the future.
+
+### manifest.json
+The web app manifest is a JSON file that tells the browser about the site and how it should behave when installed on the user's desktop or mobile device. A typical manifest file includes the app name, the icons the app should use, and the URL that should be opened when the app is launched. [more details](https://web.dev/add-manifest/).
```
Comment (stajs) [inline README.md:48]: Suggest expanding a tad for ppl not familiar with PWAs:

> The site is designed to be a [Progressive Web App](https://web.dev/what-are-pwas/) meaning that users can be prompted to install the website to their desktop or mobile device as they would a native app. The [`manifest.json`](https://web.dev/add-manifest/) specifies how it should behave when installed and typically includes the app name, the icons for the app and the URL that should be opened when the app is launched.

File: README.md [hunk 3]
```diff
@@ -33,7 +33,20 @@ Once you have your logos you can set your local variables for the navbar brand.
...
+
+Make sure you update the fields in the `manifest.json.jet` to match the sites main colours and logos, and names.
+
+Chrome has a handy dev tool called Lighthouse that can generate a report to tell you how [progressive](https://web.dev/what-are-pwas/) the site is. One of the things it checks is this manifest file to help ensure it can be installed on a device if a user wishes too.
```
Comment (stajs) [inline README.md:52]: Ah, I shoulda kept reading, see you got the link here (perhaps remove from here if using above suggestion).

Comment (gscragg) [reply README.md:52]: yeah I like your description more.

**Pull request merged**